spot-client/config.js

```diff
@@ -1,13 +1,20 @@
-/* global process */
-
 /**
- * FIXME: I wanted to keep the local environment file to place overrides for
- * development while also making the config dynamically loadable. The only way
- * I could find to do this reliably was by placing the config on a global...
+ * Overrides for the default configuration for Spot. See file default-config.js
+ * for all the default values which are used.
  */
 window.JitsiMeetSpotConfig = {
+    /**
+     * Configuration object for referencing external links and applications that
+     * provide additional support for Spot.
+     */
     ADVERTISEMENT: {
-        APP_NAME: process.env.APP_NAME
+
+        /**
+         * The name of an application that enhances the functionality of Spot.
+         *
+         * @type {string}
+         */
+        APP_NAME: undefined
```

Comment:
Yeah, at this point config.js is a template.

Reply:
Maybe this file should be left blank and people told to look at the other config file?

Reply:
I would probably include 'template' in name of the file, but I guess it can be like this for the time being. I was thinking it could be nice to have a separate directory for deployment specific things ? It feels like it's part of the source code right now